recipes/czmq/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "czmq" %}
+{% set version = "4.0.2" %}
+{% set sha256 = "794f80af7392ec8d361ad69646fc20aaa284d23fef92951334009771a732c810" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/zeromq/czmq/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  skip: True  # [win]
+  script: ./autogen.sh && ./configure && make && make install
+
+requirements:
+  build:
+    - libtool
+    - automake
+    - autoconf
+    - pkg-config
+    - zeromq 4.2.1
+  run:
+    - zeromq 4.2.1
+
+test:
```

Comment:
We normally add some file checks at a bare minimum (e.g. headers, libraries, etc.). Please take a look at [this]( https://github.com/conda-forge/zeromq-feedstock/blob/e51ab14988f5101dbb810ca902dbd7d8e239891e/recipe/meta.yaml#L57-L72 ) for some examples.